openhtf/names.py

```diff
@@ -55,5 +56,6 @@
 # Classes used in writing test scripts.
 Measurement = openhtf.util.measurements.Measurement
 OutputToJSON = openhtf.io.output.json_factory.OutputToJSON
+UploadJSON = openhtf.io.output.upload_json.UploadJSON
```

Comment:
Agreed. We should be sparing with names since we're recommending people to import * from it.